Makefile

```diff
@@ -748,7 +748,7 @@ apidocs_json = $(addprefix out/,$(apidoc_sources:.md=.json))
 
 apiassets = $(subst api_assets,api/assets,$(addprefix out/,$(wildcard doc/api_assets/*)))
 
-tools/doc/node_modules: tools/doc/package.json
+tools/doc/node_modules: tools/doc/package-lock.json
```

Comment:
Can you move this to a separate PR?

Reply:
Oops, my bad! That was meant for the eslint part...